File: Java/Day 2/Palindrome.java
import java.io.DataInputStream;

public class Palindrome {

	public static void main(String[] args) 
	{
		DataInputStream in=new DataInputStream(System.in);
		int n=0,m=0,s=0,temp=0;
		try 
		{
			System.out.print("Enter a number : ");
			n=Integer.parseInt(in.readLine());
		} catch (Exception e) {}
		temp=n;
		while(n!=0)
		{
			m=n%10;
			n=n/10;
			s=s*10+m;
		}
		if(s==temp)
			System.out.print("It is a palindromic sequence");
		else
			System.out.print("It is not a palindromic sequence");
	}

}

File: Python/Day 4/Palindrome.py
"""Palindrome"""
s=input("Enter a string: ")
s1=""
for i in range (len(s)-1,-1,-1):
    s1=s1+s[i]
if s==s1:
    print("It is a palindromic sequence")
else:
    print("It is not a palindromic sequence")